src/SegmentedString.chpl

```diff
@@ -540,24 +540,28 @@ module SegmentedString {
       var encodeArr: [0..#this.size] string; 
       var encodeOffsets: [this.offsets.aD] int;
       var encodeLengths: [this.offsets.aD] int;
-
       const lengths = this.getLengths();
       forall (i, off, len) in zip(0..#this.size, offs, lengths) {
-        // var str_entry: string;
         const filename: string = basePath+"/src/exec/%i_tmp.txt".format(i);
```

Comment:
Not a big deal, but since the filename is only used in the event `str_entry` isn't empty, we could move this into the `if` block